### PR TITLE
add beta [ios and android] configuration

### PR DESCRIPTION
**Pull request by muhzi4u**
Added beta upload [both android and ios ] configuration in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,39 @@ fastlane add_plugin cordova
...
+
+platform :android do
+  desc "Submit a new Beta Build"
+  lane :beta do
+    #gradle(task: "assembleRelease")
+     cordova(
+      platform: 'android',
+      keystore_path: './config/wheelchair88.keystore',
+      keystore_alias: 'wheelchair88',
+      keystore_password: 'Hopeti11end',
```
Comment (Almouro) [inline README.md]: you might not want to commit those in the project 😱

File: README.md [hunk 2]
```diff
@@ -17,6 +17,39 @@ fastlane add_plugin cordova
...
+      team_id:'KH5K2MDBCK'
+    )
+    # Cordova Fastlane Plugin
+    supply(apk: ENV['CORDOVA_ANDROID_RELEASE_BUILD_PATH'])
+  end 
+end
+```
+## Alpha version
```
Comment (Almouro) [inline README.md]: I feel like the lanes below were for actual production deployment, not alpha deployment, right? :\

**Comment by Almouro** [comment]
Hi @muhzi4u, thanks for the PR! :)
I added some comments and questions.

File: README.md (hunk 1 above)
Comment (muhzi4u) [inline README.md]: Sorry .It was my mistake

File: README.md (hunk 2 above)
Comment (muhzi4u) [inline README.md]: Yes.

Comment (Almouro) [inline README.md]: So in that case I think you should change the title name, right?

**Comment by Almouro** [comment]
Hi @muhzi4u, there were still some comments and some committed keystores :o
Do you want to update your PR? :)

File: README.md (hunk 2 above)
Comment (muhzi4u) [inline README.md]: yes

**Comment by muhzi4u** [comment]
Hi @Almouro  I have removed keystore name and password

File: README.md [hunk 3]
```diff
@@ -17,6 +17,38 @@ fastlane add_plugin cordova
...
 
+## Beta version
+```ruby
+platform :ios do
+  desc "Deploy new beta build"
+  lane :beta do
+    cert
+    sigh
+    gym(
+       scheme: "SCHMEA_NAME"
```
Comment (Almouro) [inline README.md]: little typo here :)

**Comment by Almouro** [comment]
Great @muhzi4u! 
I still don't understand the differences between your beta and your alpha lanes sorry :(

On Android, it's exactly the same, and on iOS, `alpha` and `beta` are deploying both on the appstore.
Alpha still deploys to the app store, so does not sound to me like an alpha deployment, and the only difference is using `testflight` or not, and using `cert +sign` vs `match`.

**Pull request closed without merging**